src/components/dashboard/account/AccountSettings.js

```diff
@@ -1,17 +1,17 @@
-import React, { Component } from 'react';
+import React, {Component} from 'react';
 
 import PaymentSettingsContainer from './PaymentSettingsContainer';
 import UserSettingsContainer from './UserSettingsContainer';
 
 class AccountSettings extends Component {
     render() {
         return (
-            <div>
+            <div className="content h-100 min-h-100 relative overflow-y-auto pt4 pb7">
```

Comment:
We need these classes, at least some of them. Without `h-100 min-h-100 overflow-y-auto` it wouldn't scroll. There might be some other way to get it to scroll on window though. This change to account settings was after the new nav changes because it wasn't scrolling.